packs/stormforge-applier-2.0.0/pack.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "stormforge-applier",
+    "displayName": "Stormforge Optimize Live Applier",
+    "layer": "addon",
+    "version": "2.0.0-beta.9",
+    "cloudTypes": [
+        "all"
+    ],
+    "charts": [
+        "charts/stormforge-applier-2.0.0-beta.9.tgz"
+    ],
+    "addonType": "monitoring",
+    "annotations": {}
```

Comment:
These packs should have [additional annotations](https://github.com/spectrocloud/pack-central/blob/a48a9c5f13f04bbfdc5d626f60fd4ed4cc397c31/packs/ambassador-6.6.0/pack.json#L5) to differentiate community vs spectrocloud managed.